.github/workflows/ci.yaml

```diff
@@ -0,0 +1,24 @@
+name: Trove Charm
+on: [pull_request, push]
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: python3 -m pip install tox
+      - name: Run linters
+        run: tox -vve lint
```

Comment:
you can simply run the existing `pep8` target, instead of creating a new one. Or at least modify the current `pep8` one. There's no point in having 2 targets doing almost the same thing. Also, make sure that the tox is passing before merging this.